docs/src/rules/require-unicode-regexp.md

```diff
@@ -98,6 +98,87 @@ function i(flags) {
 
 :::
 
+## Options
+
+This rule has one object option:
+
+* `"requireFlag": "u"|"v"` requires a particular Unicode regex flag
+
+### requireFlag: "u"
+
+The `u` flag may be preferred in environments which do not support the `v` flag.
+
+Examples of **incorrect** code for this rule with the `{ "requireFlag": "u" }` option:
+
+:::incorrect
+
+```js
+/*eslint require-unicode-regexp: ["error", { "requireFlag": "u" }] */
+
+const fooEmpty = /foo/;
+
+const fooEmptyRegexp = new RegExp('foo');
+
+const foo = /foo/v;
+
+const fooRegexp = new RegExp('foo', 'v');
+```
+
+:::
+
+Examples of **correct** code for this rule with the `{ "requireFlag": "u" }` option:
+
+:::correct
+
+```js
+/*eslint require-unicode-regexp: ["error", { "requireFlag": "u" }] */
+
+const foo = /foo/u;
+
+const fooRegexp = new RegExp('foo', 'u');
+```
+
+:::
+
+### requireFlag: "v"
+
+The `v` flag may be a better choice when it is supported because it has more
+features than the `u` flag (e.g., the ability to test Unicode properties). It
```

Comment:
```suggestion
features than the `u` flag (e.g., the ability to test Unicode properties of strings). It
```

To make the difference clear, since the `u` flag already supports properties of characters.